handler/openid/flow_refresh_token_test.go

```diff
@@ -157,6 +157,37 @@ func TestOpenIDConnectRefreshHandler_PopulateTokenEndpointResponse(t *testing.T)
 				assert.NotEmpty(t, aresp.GetExtra("id_token"))
```

Comment:
Let's check here if the hash has been set correctly

Reply:
Thanks, I will apply it.